### PR TITLE
CAR-42 - fix(carDealership): change days until termination to 14 days

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

**Comment by guilhermespopolin** [comment]
Current dependencies on/for this PR:
* main
  * **PR #3622** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3622?utm_source=stack-comment-icon" target="_blank">Graphite</a> 
    * **PR #3623** <a href="https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3623?utm_source=stack-comment-icon" target="_blank">Graphite</a>  👈

This [stack of pull requests](https://stacking.dev/?utm_source=stack-comment) is managed by [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3623?utm_source=stack-comment).

File: apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx [hunk 1]
```diff
@@ -18,8 +18,8 @@ import { LoadingSkeleton } from './LoadingSkeleton'
 import { PayForTrial } from './PayForTrial'
 import { TrialExtensionForm } from './TrialExtensionForm'
 
-// We cancel the trial after 28 days if member hasn't connected payment
-const DAYS_UNTIL_TERMINATION = 28
+// We cancel the trial after 14 days if member hasn't connected payment
```
Comment (robinandeer) [inline apps/store/src/features/carDealership/CarTrialExtensionBlock.tsx:21]:
```suggestion
// We cancel the trial if member hasn't connected payment
```

**Review by robinandeer — APPROVED** [review]
Added via Giphy

Have you seen this movie? 👆🏻

**Comment by guilhermespopolin** [comment]
> Added via Giphy [ Added via Giphy ](https://camo.githubusercontent.com/ced05275e990deb3c56322afa9a7bf41bb27d505ddcedbc466989e5581cc6a76/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f31314b586568724950537876346b2f67697068792e6769663f6369643d633934383737343373346a766e66686f386b616c72657438766d67716b6c396b7a733438686c7265686b3979617866372665703d76315f676966735f736561726368267269643d67697068792e6769662663743d67) [ ](https://camo.githubusercontent.com/ced05275e990deb3c56322afa9a7bf41bb27d505ddcedbc466989e5581cc6a76/68747470733a2f2f6d65646961302e67697068792e636f6d2f6d656469612f31314b586568724950537876346b2f67697068792e6769663f6369643d633934383737343373346a766e66686f386b616c72657438766d67716b6c396b7a733438686c7265686b3979617866372665703d76315f676966735f736561726368267269643d67697068792e6769662663743d67)
> 
> Have you seen this movie? 👆🏻

Nope. Is it good :)

**Review by robinandeer — APPROVED** [review]
@robinandeer approved this PR from Slack with [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/3623)

**Pull request merged**